### ISSUE TITLE
Can turnOffAfterms set accessory state directly?

### ISSUE THREAD

**Issue by jdtsmith** [arachnetech/homebridge-mqttthing#87]
Thanks for a great plugin.  I wonder if it's possible for `turnOffAfterms` to set the accessory characteristic for a switch to `Off` even in the absence of a `setOn` topic?  My use case is an MQTT-enabled relay unit that publishes long presses on a separate topic from normal switch use, but there is no equivalent set topic the device understands.  The device just is _reporting_ the long press.  I want to use the long press in automations, but there is no MQTT-based way to turn it back off after the long press is triggered.  A stateless dummy doesn't work here either, since you can't use that in complex automations.

**Comment by jdtsmith** [comment]
In fact, I just found out that you have already implemented this, but hadn't yet hooked it up!  I made this simple change:

```
     // Characteristic.On
     function characteristic_On(service) {
-        booleanCharacteristic(service, 'on', Characteristic.On, config.topics.setOn, config.topics.getOn, null, null, config.turnOffAfterms);
+      booleanCharacteristic(service, 'on', Characteristic.On, config.topics.setOn, config.topics.getOn, null, null, config.turnOffAfterms,config.resetStateAfterms);
     }
```

and I added a config `resetStateAfterms` and that did the trick immediately.  You might consider turning on the capability you already have in there!

**Comment by arachnetech** [comment]
Yes, I'll make that change so that you can use switches like this - good idea.

That `resetStateAfterms` in `booleanCharacteristic` is used by the motion sensor type to support turning off in just this scenario (as sensors don't have set topics), but I appear to have forgotten to document this. I'll fix that too.

**Comment by mientki** [comment]
Did it also work with LeakSensor? My sensor only sends one RF code. I would like the free state of water detection to expire after a defined, defined time, e.g. 60 sec.

**Comment by arachnetech** [comment]
Implemented (also with leakSensor) in 1.0.32.
